docs/v3.x/concepts/webhooks.md

```diff
@@ -54,16 +54,25 @@ You can delete a webhook by clicking on the `trash` icon.
 
 By default Strapi webhooks can be triggered by the following events:
 
-| Name           | Description                                     |
-| -------------- | ----------------------------------------------- |
-| `entry.create` | Triggered when a Content Type entry is created. |
-| `entry.update` | Triggered when a Content Type entry is updated. |
-| `entry.delete` | Triggered when a Content Type entry is deleted. |
-| `media.create` | Triggered when a media is created.              |
-| `media.delete` | Triggered when a media is deleted.              |
+| Name              | Description                                           |
+| ----------------- | ----------------------------------------------------- |
+| `entry.create`    | Triggered when a Content Type entry is created.       |
+| `entry.update`    | Triggered when a Content Type entry is updated.       |
+| `entry.delete`    | Triggered when a Content Type entry is deleted.       |
+| `entry.publish`   | Triggered when a Content Type entry is published.\*   |
```

Comment:
What's the purpose of these `\*`?

Reply:
I don't know why there is the `\`, I only put `*`, some autoformatting changed that during the commit I think.
It's just an asterix :)